ibm_catalog.json

```diff
@@ -468,6 +551,7 @@
             },
             {
               "key": "management_endpoint_type_for_bucket",
+              "hidden": true,
```

Comment:
This should not be hidden anymore - and needs to be exposed in security enforced variation but only allow private or direct in the dropdown